packages/display_renderers/config/robot_info_renderer_node/default.yaml

```diff
@@ -0,0 +1 @@
+frequency: 0.1
```

Comment:
Fragments have a default TTL of `10s`, this will create a race condition. You can keep this at 0.1Hz as long as you increase the TTL. Solution proposed down below.